dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/impl/CustomCSSProcessor.java

```diff
@@ -0,0 +1,291 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
```

Comment:
2018?

Reply:
Can you please change it across the PR?

Reply:
It's not new code. This class and the others in this PR comes from the openapi-3.0 feature with some minor changes.